File: C++/Estrutura Repetitiva/Enquanto/Crescente.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    int x, y;

    cout << "Digite dois numeros: " << endl;
    cin >> x;
    cin >> y;

    while (x != y)
    {
        if (x > y)
        {
            cout << "DECRESENTE! " << endl;
        }
        else
        {
            cout << "CRESENTE!" << endl;
        }
        cout << "Digite outros dois numeros: " << endl;
        cin >> x;
        cin >> y;
    }

    return 0;
}

File: Python/Estrutura Repetitiva/Enquanto/Crescente.py
x: int; y: int

print(f"Digite dois numeros: ")
x = int(input())
y = int(input())

while x != y:
    if x < y:
        print(f"CRESCENTE !")
    
    else:
        print(f"DECRESCEMTE !")
    
    print(f"Digite outros dois numeros: ")
    x = int(input())
    y = int(input())

